packages/k-ramel/doc/REACTIONS.md

```diff
@@ -16,3 +16,66 @@
     <a href="./STORE.md">store</a>
   &nbsp;]
 </p>
+
+## What is a reaction?
+A reaction is something your plug to a listeners (see [listeners' documentation here](./LISTENERS.md)).\
+This is where you write your logical code.\
+In a reaction you can access:
+ - the action that trigger your reaction (the action that your listener has filtered)
+ - the store (the k-ramel store, with dispatch, data, drivers, etc)
+ - the drivers
+
+A reaction is a function whom the signature is: `(action, store, drivers)`.
+
+## Example & API
+```js
+import { when } from 'k-ramel'
+
+// a reacton is a function
+const reaction = () => {
+  console.log('I am a reaction!')
+}
+
+// you can acces the action that triggered your reaction
+const reaction = (action) => {
+  const { type, payload } = action
+
+  console.log('I am triggered by the action:', type)
+  console.log('\tthe payload is:', payload)
+}
+
+// you can also acces the store, to read the state, or modify it, or dispatch a new event, etc
```

Comment:
// you can also **access** the store, to read the state, or modify it, or dispatch a new event, etc